qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,1037 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Class to store and manage the results of calibration experiments."""
+
+import os
+from collections import namedtuple, defaultdict
+from datetime import datetime
+from typing import Any, Dict, Set, Tuple, Union, List, Optional
+import csv
+import dataclasses
+import warnings
+import regex as re
+
+from qiskit.pulse import (
+    ScheduleBlock,
+    DriveChannel,
+    ControlChannel,
+    MeasureChannel,
+    Call,
+    Instruction,
+    AcquireChannel,
+    RegisterSlot,
+    MemorySlot,
+    Schedule,
+)
+from qiskit.pulse.channels import PulseChannel
+from qiskit.circuit import Parameter, ParameterExpression
+from qiskit_experiments.calibration.exceptions import CalibrationError
+from qiskit_experiments.calibration.parameter_value import ParameterValue
+
+ParameterKey = namedtuple("ParameterKey", ["parameter", "qubits", "schedule"])
+ScheduleKey = namedtuple("ScheduleKey", ["schedule", "qubits"])
+ParameterValueType = Union["ParameterExpression", float, int, complex]
```

Comment:
Why we cannot import this from `qiskit.circuit.parameterexpressions`?

Reply:
No valid reason. See here: 03f2d7b